libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/diff/DiffCache.kt

```diff
@@ -24,6 +24,7 @@ interface MutableDiffCache<E> : DiffCache<E> {
     fun removeAt(index: Int): E?
     fun add(index: Int, element: E?)
     operator fun set(index: Int, element: E?)
+    fun clear()
```

Comment:
Not used?

Reply:
It's a leftover from previous tests that I forgot to remove, thanks!